app/_hub/kong-inc/rate-limiting-advanced/how-to/_index.md

```diff
@@ -0,0 +1,593 @@
+                "tags": null
+            }
+        ]
+    }
+    ```
+
+1. Using the username or ID of the group (`Gold` in this example),
+remove the consumer from the group:
+
+    ```bash
+    curl -i -X DELETE http://{HOSTNAME}:8001/consumers/Amal/consumer_groups/Gold
+    ```
+
+    If successful, you receive the following response:
+    ```
+    HTTP/1.1 204 No Content
+    ```
+
+1. To verify, check the consumer object configuration:
+
+    ```bash
+    curl -i -X GET http://{HOSTNAME}:8001/consumer_groups/Gold
+    ```
+
+    Response, with no consumers assigned:
+
+    ```json
+    {
+        "consumer_group": {
+            "created_at": 1638917780,
+            "id": "be4bcfca-b1df-4fac-83cc-5cf6774bf48e",
+            "name": "Gold",
+            "tags": null
+        }
+    }
+    ```
+
+## Delete consumer group configurations
+
+You can also clear the configuration of a consumer group without deleting the consumer group itself.
+
+With this method, the consumers in the group aren't deleted and are still in the consumer group.
+
+1. Delete the consumer group configuration using the following request:
+
+    ```bash
+    curl -i -X DELETE http://{HOSTNAME}:8001/consumer_groups/Gold/overrides/plugins/rate-limiting-advanced
+    ```
+
+    If successful, you receive see the following response:
+    ```
+    HTTP/1.1 204 No Content
+    ```
+    
+1. To verify, check the consumer object configuration:
+
+    ```bash
+    curl -i -X GET http://{HOSTNAME}:8001/consumer_groups/Gold
+    ```
+
+    Response, without a `plugins` object:
+
+    ```json
+    {
+        "consumer_group": {
+            "created_at": 1638917780,
+            "id": "be4bcfca-b1df-4fac-83cc-5cf6774bf48e",
+            "name": "Gold",
+            "tags": null
+        }
+    }
+    ```
+
+
+## Delete consumer group
+
+If you don't need a consumer group anymore, you can delete it. This removes
+all consumers from the group, and deletes the group itself. The consumers in
+the group are not deleted.
+
+1. Delete a consumer group using the following request:
+
+    ```bash
+    curl -i -X DELETE http://{HOSTNAME}:8001/consumer_groups/Gold
+    ```
+
+    If successful, you receive see the following response:
+    ```
+    HTTP/1.1 204 No Content
+    ```
+
+1. Check the list of consumer groups to verify that the `Gold` group is gone:
+
+    ```bash
+    curl -i -X GET http://{HOSTNAME}:8001/consumer_groups
+    ```
+
+    Response:
+    ```json
+    {
+    "data": [],
+    "next": null
+    }
+    ```
+
+1. Check a consumer that was in the group to make sure it still exists:
+
+    ```bash
+    curl -i -X GET http://{HOSTNAME}:8001/consumers/Amal
+    ```
+
+    An `HTTP/1.1 200 OK` response means the consumer exists.
+
+
+## Manage multiple consumers
+
+You can perform many `/consumer_groups` operations in bulk.
+
+1. Assuming you deleted the group `Gold` in the previous section, create it again,
+along with another group named `Speedsters`:
```

Comment:
```suggestion
1. Assuming you deleted the group `Gold` in the previous section, create it again,
along with another group named `Silver`:
```
Going along with Diana's comment about "Barry Allen", let's switch all of these to plain terms.

* Replace all "Speedsters" with "Silver"
* Replace all BarryAllen and WallyWest with more generic names